WPF app with BlazorWebView fails to compile the Razor components

### Description

I created a WPF app with a BlazorWebView and then tried to add the code for the default Blazor template, but I get a compilation error saying that one of the namespaces is missing, even though I have components in the Shared folder:

error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf'

Here's my code: https://github.com/danroth27/BlazorHybridWpf

### Steps to Reproduce

1. Clone and build https://github.com/danroth27/BlazorHybridWpf

### Version with bug

Preview 10 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Win11

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

```
1>------ Rebuild All started: Project: BlazorHybridWpf, Configuration: Debug Any CPU ------
Restored C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\BlazorHybridWpf.csproj (in 22 ms).
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\Pages\Index.razor(9,1,9,56): warning RZ10012: Found markup element with unexpected name 'SurveyPrompt'. If this is intended to be a component, add a @using directive for its namespace.
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\_Imports.razor(9,23,9,29): error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf' (are you missing an assembly reference?)
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\_Imports.razor(9,23,9,29): error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf' (are you missing an assembly reference?)
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\_Imports.razor(9,23,9,29): error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf' (are you missing an assembly reference?)
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\_Imports.razor(9,23,9,29): error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf' (are you missing an assembly reference?)
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\_Imports.razor(9,23,9,29): error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf' (are you missing an assembly reference?)
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\_Imports.razor(9,23,9,29): error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf' (are you missing an assembly reference?)
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\_Imports.razor(9,23,9,29): error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf' (are you missing an assembly reference?)
1>C:\Users\user\source\repos\BlazorHybridWpf\BlazorHybridWpf\_Imports.razor(9,23,9,29): error CS0234: The type or namespace name 'Shared' does not exist in the namespace 'BlazorHybridWpf' (are you missing an assembly reference?)
1>Done building project "BlazorHybridWpf_n5hqffra_wpftmp.csproj" -- FAILED.
```

--- a/src/BlazorWebView/samples/BlazorWpfApp/MainWindow.xaml.cs
+++ b/src/BlazorWebView/samples/BlazorWpfApp/MainWindow.xaml.cs
@@ -45,8 +45,4 @@ namespace BlazorWpfApp
 			blazorWebView1.WebView.CoreWebView2.ExecuteScriptAsync("alert('hello from native UI')");
 		}
 	}
-
-	// Workaround for compiler error "error MC3050: Cannot find the type 'local:Main'"
-	// It seems that, although WPF's design-time build can see Razor components, its runtime build cannot.
-	public partial class Main { }
 }

--- a/src/BlazorWebView/src/Wpf/build/Microsoft.AspNetCore.Components.WebView.Wpf.targets
+++ b/src/BlazorWebView/src/Wpf/build/Microsoft.AspNetCore.Components.WebView.Wpf.targets
@@ -6,7 +6,7 @@
     <CoreCompileDependsOn Condition="'$(PublishProtocol)' == 'ClickOnce' or '$(PublishProtocol)' == 'FileSystem'">$(CoreCompileDependsOn);StaticWebAssetsPrepareForRun</CoreCompileDependsOn>
 
     <!-- Workaround for https://github.com/dotnet/wpf/issues/5697 (fixes https://github.com/dotnet/maui/issues/3526) -->
-    <RootNamespace Condition="$(UseWPF) == 'true' AND $(RootNamespace.EndsWith('_wpftmp'))">$(_TargetAssemblyProjectName)</RootNamespace>
+    <RootNamespace Condition="'$(UseWPF)' == 'true' AND $(RootNamespace.EndsWith('_wpftmp'))">$(_TargetAssemblyProjectName)</RootNamespace>
   </PropertyGroup>
 
   <Target Name="AddStaticWebAssetsForClickOnce" AfterTargets="ComputeFilesToPublish" Condition="'$(PublishProtocol)' == 'ClickOnce'">

--- a/src/BlazorWebView/src/Wpf/build/Microsoft.AspNetCore.Components.WebView.Wpf.targets
+++ b/src/BlazorWebView/src/Wpf/build/Microsoft.AspNetCore.Components.WebView.Wpf.targets
@@ -4,6 +4,9 @@
     <StaticWebAssetBasePath>/</StaticWebAssetBasePath>
     <StaticWebAssetProjectMode>Root</StaticWebAssetProjectMode>
     <CoreCompileDependsOn Condition="'$(PublishProtocol)' == 'ClickOnce' or '$(PublishProtocol)' == 'FileSystem'">$(CoreCompileDependsOn);StaticWebAssetsPrepareForRun</CoreCompileDependsOn>
+
+    <!-- Workaround for https://github.com/dotnet/wpf/issues/5697 (fixes https://github.com/dotnet/maui/issues/3526) -->
+    <RootNamespace Condition="$(UseWPF) == 'true' AND $(RootNamespace.EndsWith('_wpftmp'))">$(_TargetAssemblyProjectName)</RootNamespace>
   </PropertyGroup>
 
   <Target Name="AddStaticWebAssetsForClickOnce" AfterTargets="ComputeFilesToPublish" Condition="'$(PublishProtocol)' == 'ClickOnce'">

